guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -43,7 +43,7 @@ Use the following layout instructions to customize your layout:
 
 Defines a block.
 
-<p><b>Details:</b> A block is a unit of page output that renders some distinctive content (anything vvisually tangible for the end-user), such as a piece of information or a user interface element.
+<p><b>Details:</b> A block is a unit of page output that renders some distinctive content (anything visually tangible for the end-user), such as a piece of information or a user interface element.
```

Comment:
Please also change this in v2.1 version of the page.

Reply:
@shrielenee I have already applied above changes in v2.1 https://github.com/magento/devdocs/pull/2130/commits/2c2d39c07d76d6a1c137308650db2a84958a2a05

Reply:
Hello @shrielenee How can I merge both branch in one ? Is there any documentation for merge PRs. I have tried to switch my branch "hitesh-wagento:patch-8" but it's throwing error "error: pathspec 'hitesh-wagento:patch-8' did not match any file(s) known to git."